guides/Collaborating with Kiln.md

```
@@ -2,9 +2,9 @@
 
 ---
 
-# This guide as moved to [our docs](https://docs.getkiln.ai/docs/collaboration). This page may be out of date.
+# This guide as moved to [our docs](https://docs.kiln.tech/docs/collaboration). This page may be out of date.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)